### DISCUSSION TITLE
can I use  apisix  as  a  k8s  Ingress controller

### DISCUSSION THREAD

**Discussion by xianshun163** [apache/apisix#5457] [Q&A] [answered]
can I use  apisix  as  a  k8s  Ingress controller. 
where can I  find  some  document  about it.

**Comment by tokers** [comment, marked as answer]
Sure, please checkout the following links:

1. https://github.com/apache/apisix-ingress-controller
2. https://apisix.apache.org/docs/ingress-controller/getting-started/
